### PR TITLE
document for HTMLAreaElement.ping

### PR DESCRIPTION
**Pull request by ShubhamOulkar**
This PR add documentation for HTMLAreaElement.ping property.

It is part of the discussion to document all interoperable features ([mdn/discussions#476](https://github.com/orgs/mdn/discussions/476))

Resource
[ping](https://html.spec.whatwg.org/multipage/image-maps.html#dom-area-ping)

### CODE REVIEW

File: files/en-us/web/api/htmlareaelement/ping/index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+title: "HTMLAreaElement: ping property"
+short-title: ping
+slug: Web/API/HTMLAreaElement/ping
+page-type: web-api-instance-property
+browser-compat: api.HTMLAreaElement.ping
+---
+
+{{ApiRef("HTML DOM")}}
+
+The **`ping`** property of the {{domxref("HTMLAreaElement")}} interface is a space-separated list of URLs.
```
Comment (skyclouds2001) [inline files/en-us/web/api/htmlareaelement/ping/index.md:11]:
```suggestion
The **`ping`** property of the {{domxref("HTMLAreaElement")}} interface is a space-separated list of URLs that will be used to notify if the user follows the hyperlink.
```

the properties list also need update

File: files/en-us/web/api/htmlareaelement/ping/index.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+> **Note:** This property is not effective in Firefox and its usage may be limited due to privacy and security concerns.
+
+## Example
+
+```html
+<map>
+  <area
+    href="https://example.com"
+    ping="https://example-tracking.com"
```
Comment (skyclouds2001) [inline files/en-us/web/api/htmlareaelement/ping/index.md:23]: see https://github.com/mdn/content/pull/31904#discussion_r1466395020

File: files/en-us/web/api/htmlareaelement/ping/index.md (hunk 1 above)
Comment (ShubhamOulkar) [reply files/en-us/web/api/htmlareaelement/ping/index.md:11]: Can we write like this
```suggestion
The **`ping`** property of the {{domxref("HTMLAreaElement")}} interface is a space-separated list of URLs that will be used by browser to send PING object.
```

File: files/en-us/web/api/htmlareaelement/ping/index.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+title: "HTMLAreaElement: ping property"
+short-title: ping
+slug: Web/API/HTMLAreaElement/ping
+page-type: web-api-instance-property
+browser-compat: api.HTMLAreaElement.ping
+---
+
+{{ApiRef("HTML DOM")}}
+
+The **`ping`** property of the {{domxref("HTMLAreaElement")}} interface is a space-separated list of URLs. When the link is followed, the browser will send POST requests with the body PING to the URLs.
```
Comment (skyclouds2001) [inline files/en-us/web/api/htmlareaelement/ping/index.md:11]:
```suggestion
The **`ping`** property of the {{domxref("HTMLAreaElement")}} interface is a space-separated list of URLs. When the link is followed, the browser will send {{HTTPMethod("POST")}} requests with the body PING to the URLs.
```

File: files/en-us/web/api/htmlareaelement/index.md [hunk 4]
```diff
@@ -37,6 +37,8 @@ _Inherits properties from its parent {{domxref("HTMLElement")}}._
   - : A string containing the password specified before the domain name.
 - {{domxref("HTMLAreaElement.pathname")}}
   - : A string containing the path name component, if any, of the referenced URL.
+- {{domxref("HTMLAreaElement.ping")}}
+  - : A space-separated list of URLs. When the link is followed, the browser will send POST requests with the body PING to the URLs.
```
Comment (skyclouds2001) [inline files/en-us/web/api/htmlareaelement/index.md:41]:
```suggestion
- {{domxref("HTMLAreaElement.ping")}}
  - : A space-separated list of URLs. When the link is followed, the browser will send {{HTTPMethod("POST")}} requests with the body PING to the URLs.
```

Comment (ShubhamOulkar) [reply files/en-us/web/api/htmlareaelement/index.md]: Thank you 🙌

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**